lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require 'twilio'
+
+class Order
+  include Twilio
+  attr_reader :current_order
+
+  def initialize
+    @current_order = Hash.new(0)
+  end
+
+  def add(order)
+    order.each_pair do |item, quantity|
+      @current_order[item] += quantity
+    end
+  end
+
+  def calculate
```

Comment:
Is there a way you can make this method shorter?  Ideally 5 lines or less (including `def` and `end`).